fred2/missionsave.cpp

```diff
@@ -1635,6 +1635,128 @@ int CFred_mission_save::save_common_object_data(object *objp, ship *shipp)
 	return err;
 }
 
+int CFred_mission_save::save_containers()
+{
+	if (Mission_save_format == FSO_FORMAT_RETAIL) {
+		return 0;
+	}
+
+	if (Sexp_containers.empty()) {
+		fso_comment_pop(true);
+		return 0;
+	}
+
+	required_string_fred("#Sexp_containers");
+	parse_comments(2);
+
+	bool list_found = false;
+	bool map_found = false;
```

Comment:
These local vars are required because lists appear in the mission file before maps. I guess I could mention that in a comment.